app/main.py

```diff
@@ -1,7 +1,47 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+
+WEATHER_API_URL = "http://api.weatherapi.com/v1/current.json"
+DEFAULT_CITY = "Paris"
+
+
+def fetch_weather_data(api_key: str, city: str) -> dict:
+    try:
+        response = requests.get(
+            WEATHER_API_URL, params={"key": api_key, "q": city}
+        )
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.RequestException as e:
+        print(f"Network error: {e}")
```

Comment:
It is a good practice to log errors instead of printing them. This way, you can have a more detailed record of what went wrong and when. You might consider using the `logging` module here.